runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -247,6 +249,10 @@ protected void startUp()
 				{
 					// Debug config on context needs to be set before .makeCurrent call
 					glContext.enableGLDebugMessage(true);
+
+					// 0x826b = severity Unknown
+					glContext.glDebugMessageControl(GL_DEBUG_SOURCE_API, GL_DEBUG_TYPE_OTHER,
```

Comment:
Isnt there constant for severity Unknown in the GL api somewhere? Also, you can access the constants from gl. object directly, no need to import them

Reply:
https://jogamp.org/wiki/index.php/GL_Debug_Output based on this there should be constants for severity

Reply:
No. https://i.imgur.com/WFoGVr7.png